gearcatalog/frontend/src/components/accounts/Login.tsx

```diff
@@ -64,7 +64,7 @@ const Login = ({ isAuthenticated, login }: LoginProps) => {
               </button>
             </div>
             <p>
-              Don&apo;t have an account? <Link to="/register">Register</Link>
+              Don&apos;t have an account? <Link to="/register">Register</Link>
```

Comment:
In React/JSX/TSX do you need to use HTML elements like this? Does it not convert `'` to `&apos;` for you?

Reply:
Apparently it does convert them for you. My linter was yelling at me about it so I changed it. I will change it back.